src/content/en/2020/javascript.md

```diff
@@ -12,19 +11,20 @@ tkadlec_bio: Tim is a web performance consultant and trainer focused on building
 discuss: 2038
 results: https://docs.google.com/spreadsheets/d/1cgXJrFH02SHPKDGD0AelaXAdB3UI7PIb5dlS0dxVtfY/
 queries: 02_JavaScript
-#featured_quote: TODO
+featured_quote: JavaScript has come a long way from its humble origins as the last of the three web cornerstones—alongside CSS and HTML. Today, JavaScript has started to infiltrate a broad spectrum of the technical stack. JavaScript is no longer confined to the client-side—it's an increasingly popular choice for build tools and server-side scripting, and is creeping its way into the CDN layer as well thanks to edge computing solutions.
```

Comment:
* Adding space around `—` in `... client-side — it's an ...`. Alternative, this can be split into two sentences.
* Should the `thanks to edge computing solutions` phrase be put in brackets separated by a comma or something?

```suggestion
featured_quote: JavaScript has come a long way from its humble origins as the last of the three web cornerstones—alongside CSS and HTML. Today, JavaScript has started to infiltrate a broad spectrum of the technical stack. JavaScript is no longer confined to the client-side — it's an increasingly popular choice for build tools and server-side scripting, and is creeping its way into the CDN layer as well thanks to edge computing solutions.
```

Reply:
We haven't been doing that in other chapters so would prefer to keep consistent.

Reply:
The [style guide we are using says no spaces around em-dash](https://developers.google.com/style/dashes).

However I do agree it looks weird here so close to a hyphenated word.

Also I think this sentence is rather long with a first part, the em-dash, and then the third part. And the two "ands" don't help (though I agree technically correct).

I think we should rephrase. What about something like this?

```suggestion
featured_quote: JavaScript has come a long way from its humble origins as the last of the three web cornerstones—alongside CSS and HTML. Today, JavaScript has started to infiltrate a broad spectrum of the technical stack–it is no longer confined to the client-side and it's an increasingly popular choice for build tools and server-side scripting. JavaScript is also creeping its way into the CDN layer as well thanks to edge computing solutions.
```

Though not made about three sentences basically all starting with the same word (JavaScript), but that's probably just getting picky.

Reply:
Look good to me, but I think we should even split it at `stack–it`. That is literally a hyphenated word now, which doesn't make sense.

Reply:
Works for me, and also means we have a 4th sentence that doesn't begin with JavaScript 🙂

```suggestion
featured_quote: featured_quote: JavaScript has come a long way from its humble origins as the last of the three web cornerstones—alongside CSS and HTML. Today, JavaScript has started to infiltrate a broad spectrum of the technical stack. It is no longer confined to the client-side and it's an increasingly popular choice for build tools and server-side scripting. JavaScript is also creeping its way into the CDN layer as well thanks to edge computing solutions.
```

Reply:
Thoughts/suggestions @tkadlec?